src/test/ipv-stub/integration/ipv-user-identity-response.test.ts

```diff
@@ -0,0 +1,92 @@
+import supertest from "supertest";
```

Comment:
I think this file should be called `ipv-user-identity.test.ts` and should be in the same folder as the ipv authorize integration test